### ISSUE TITLE
NTR: Flank mass (Endocrine & genitourinary)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7956]
New term request Flank mass (elthat_100529022046):   UMLS: 0744049

**Comment by pnrobinson** [comment]
@MickeySegal
I am not entirely sure if this is well delineated from [Abdominal mass HP:0031500](https://hpo.jax.org/app/browse/term/HP:0031500)?

**Comment by MickeySegal** [comment]
https://www.merriam-webster.com/dictionary/flank
"the fleshy part of the side between the ribs and the hip"

A flank mass is often a sign of kidney disease rather than abdominal disease, e.g., 
https://www.ncbi.nlm.nih.gov/books/NBK1326/
"The presence of bilateral palpable flank masses in infants"

**Comment by pnrobinson** [comment]
@MickeySegal 
I think this is the first HPO term with flank and it is a little hard to place it in the hierarchy. I have "parked" the term under abdomen for now, but this can be refined later.

Adding new term: 
- ID: HP:6000363
- label: Flank mass
- parents: [Abnormal abdomen morphology (HP:0001438)](https://hpo.jax.org/app/browse/term/HP:0001438)
- pmid: 
- def: An abnormal enlargement or swelling in the flank, which is the fleshy part of the side of the body between the ribs and the hip.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7956
- synonyms:
